src/components/User/UserAvatar.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+
+import { PreviewCamera, PreviewEmote } from '@dcl/schemas'
+import { WearablePreview } from 'decentraland-ui'
```

Comment:
I think this import is going to fail at build time

Reply:
You are right, it does:
>>>   WebpackError: ReferenceError: window is not defined

Reply:
Fixed :)